src/streamlink/plugins/youtube.py

```diff
@@ -203,14 +207,29 @@ def _create_adaptive_streams(self, info, streams, protected):
         return streams, protected
 
     def _find_channel_video(self):
-        res = http.get(self.url)
-        match = _channelid_re.search(res.text)
-        if not match:
-            return
+        res = http.get(self.url, headers={"User-Agent": useragents.CHROME})
```

Comment:
could you also force the **chrome header on the whole plugin** and remove all the other headers

if the youtube stream is embedded,
the User-Agent can change to something else from the previous plugin and might break some regex

```diff
     def _get_streams(self):
+        http.headers.update({'User-Agent': useragents.CHROME})
```